features/keychain/module/__tests__/schnorr.test.js

```diff
@@ -0,0 +1,51 @@
+import { create } from '../crypto/secp256k1'
+
+const fixtures = [
+  {
+    priv: '90de83eea26049afc40ba7d13fd8d4537331cd226f17051c97ca56c696af66b5',
+    pub: '02c24e41b8ec4d091f9bfbb481fde7ce0808ed820db8e93409cc404da8b9de7e92',
+    pub2: '039c6d4d5e46fd14b3685e513dab7a34090a2a862bd23e122421ce5bc59d754276',
+    buffer: '52be7b43a029336afb5bca87f33b5cbe1a84d70e321db62dc12c14eac3c8b3a3',
+    entropy: '1230000000000000000000000000000000000000000000000000000000000000',
+    sig: 'dbbe008a2352d84bf86ba7db400b9d0927be174820ffcd8c45d4e52a416d6130af75ac72e1d8399ca2b5334d76ad93cea370d7d6649a3072ce3c34da567517f6',
+  },
```

Comment:
@feri42 where'd we get these fixtures?

Reply:
Right. I console logged in the existing taproot tests. Not sure all of them exist as strings in the asset tests.